coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -619,6 +619,7 @@ def _get_tensor_shape_from_type(self, type_):
                 assert ashape.get_shape() == element_shape
             shape = [-1] + list(element_shape)
         else:
+            import pdb;pdb.set_trace()
```

Comment:
You probably want to remove this line.

Reply:
@TobyRoseman san,

Thank you for your review!
I removed it.